### PR TITLE
feat(TrianglaOutput): add triangle output (node, ele, neigh)

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: src/geode/io/mesh/triangle_output.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    }
+} // namespace
+
+namespace geode
+{
+    namespace detail
+    {
+        void TriangleOutput::write() const
+        {
+            const auto file = filename().substr(
```
Comment (panquez) [inline src/geode/io/mesh/triangle_output.cpp:98]: string_view ?

File: src/geode/io/mesh/triangle_output.cpp [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            {
+                ele << SPACE << mesh.polygon_vertex( { p, v } );
+            }
+            ele << EOL;
+        }
+        ele << std::flush;
+    }
+
+    void write_neigh(
+        const std::string& filename, const geode::TriangulatedSurface2D& mesh )
```
Comment (panquez) [inline src/geode/io/mesh/triangle_output.cpp:67]: string_view everywhere ?

**Review by panquez — APPROVED** [review]
Be careful to typo in commit name

File: src/geode/io/mesh/triangle_output.cpp (hunk 2 above)
Comment (BotellaA) [reply src/geode/io/mesh/triangle_output.cpp:67]: StrCat returns a std::string.

**Pull request merged**